CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ FEATURES:
 
 IMPROVEMENTS:
  * cli: Improved `scaling policy` commands with -verbose, auto-completion, and prefix-matching [[GH-9964](https://github.com/hashicorp/nomad/issues/9964)]
+ * cli: Added optional `-task <task-name>` flag to `alloc logs` to match `alloc exec` [[GH-10026](https://github.com/hashicorp/nomad/issues/10026)]
```

Comment:
_Super_ nitpick that should totally be something we fix with `make changelogfmt`: we put the shorter lines for each component first.